### DISCUSSION TITLE
How to add an option for rechargeable to a device?

### DISCUSSION THREAD

**Discussion by robchandhok** [andrew-codechimp/HA-Battery-Notes#2537] [General]
Love Battery Notes!

I just noticed someone added entries for Hunter Douglas shades.  We happen to have some "Silhouette" model ones, that we recently changed from battery packs to rechargeable packs.

Battery notes has an entry for the 12AA variant battery packs.  I thought I could add another entry for the rechargeable and it would let me choose when adding the new device, but I can't seem to figure out the right trick.

Is this something that works that I don't get, or a new feature request to allow different battery configurations for a {manufacturer,model} ?

Thank you!

What I thought I could add:

```
        {
            "manufacturer": "Hunter Douglas",
            "model": "Silhouette",
            "battery_type": "Rechargeable"
        },


```

**Comment by andrew-codechimp** [comment]
Thanks for pointing this out, devices that can have multiple battery types should be marked as MANUAL so that Battery Notes does not suggest it, I have just changed this.
There's only a few devices like this and implementing a choice would be a lot of effort.
